src/browser_app_skeleton/webpack.mix.js

```diff
@@ -46,7 +46,7 @@ mix
   // https://github.com/JeffreyWay/laravel-mix/blob/master/docs/mixjs.md
   .js("src/js/app.js", "public/js")
   // SASS entry file. Uses autoprefixer automatically.
-  .sass("src/css/app.scss", "public/css")
+  .sass("src/css/app.scss", "css")
```

Comment:
This still outputs to the `public` folder, but seems to assume that namespace now.

Leaving this in its older form yields files in `public/public/css`

Reply:
What about the `js` one? That still says `public/js` 🤔 . Also, is there a config for this? If so, maybe we can expose the config by default and set it to public. Then people won't have to go digging through docs to change that (if they needed).

Reply:
oh wait, line 60 says `.setPublicPath("public")`, so maybe that's where it's configured. If that's so, then I think just the js part needs to be updated, right?

Reply:
Weirdly enough, the JS part outputs correctly! I think moving that `setPublicPath` up will work, though, so I'll run some tests and re-submit so that it's a bit easier to reason about.

Reply:
Okay, tested this locally and it worked when I moved up the `setPublicPath` call! Good catch!